web/src/styles/app/table.css

```diff
@@ -146,4 +150,5 @@
 .table-frozen-narrow-header .table-frozen-data tr > th:first-child,
 .table-frozen-narrow-header .table-frozen-data tr > td:first-child {
   min-width: 90px;
+  width: 90px;
```

Comment:
I honestly can't remember why I needed this.  But if I didn't have it, the hidden columns were too wide.  They'd just explode in size, and the second column would be hidden by it forever.  It was weird.